components/ai_chat/resources/page/state/conversation_context.tsx

```diff
@@ -657,7 +659,13 @@ export function ConversationContextProvider(props: React.PropsWithChildren) {
       (url?: Url) => setPartialContext({ generatedUrlToBeOpened: url }),
     setIgnoreExternalLinkWarning,
     disassociateContent,
-    associateDefaultContent,
+          associateDefaultContent,
+      processDroppedImages: (images: Mojom.UploadedFile[]) => {
+        setPartialContext({
+          isUploadingFiles: true
+        })
+        processUploadedImage(images)
```

Comment:
do we need to set `isUploadingFiles` to `false` again when this promise resolves?

Reply:
We set it already in `processUploadedImage` but I handled the case there are 0 files. Not sure if that can happen though. b26fe8c73dcd7988cb00c0347fff38064d96a000